### PR TITLE
Migration/customisation for glosses

### PR DESCRIPTION
**Pull request by NavishkaJayasinghe**
Following are the list of changes made to the Signbank code.

1) Added hint column for glosses. (https://ackama.atlassian.net/browse/N2-3)

2) Added a new signer table and a signer column for glosses. (https://ackama.atlassian.net/browse/N2-5)

3) Corrected the failing tests cases.

4) Changed the gloss translation behaviour in Signbank. Originally a gloss had its translations (only one). But in Freelex, a gloss has 3 translations namely glossmain, glosssecondary and glossminor. Therefore, the Signbank was changed to have all 3 types of translations throughout Signbank app. This includes changes to the model, public ui and admin ui. (https://ackama.atlassian.net/browse/N2-25)

### CODE REVIEW

File: signbank/dictionary/models.py [hunk 1]
```diff
@@ -630,6 +644,21 @@ def __str__(self):
         return str(self.morpheme.idgloss) + ' is ' + str(self.role) + ' of ' + str(self.parent_gloss.idgloss)
 
 
+@python_2_unicode_compatible
+class Signer(models.Model):
+    """The list of signers"""
+    #: Signer name.
+    name = models.CharField(max_length=150, unique=False)
```
Comment (joshmcarthur) [inline signbank/dictionary/models.py]: Should the signer's name be unique to reduce duplicates?

**Review by joshmcarthur — COMMENTED** [review]
Can you please update this pull request description to:

* Include a reference to the Jira ticket(s) that it implements or partially implements
* Include a short summary of the change
* Screenshots of the new UI introduced

Ackama has some recommendations for how PRs should be written. These can be found in our [Confluence](https://ackama.atlassian.net/wiki/spaces/DEV/pages/26560430682/Pull+Request+Review#Writing-your-PR)

The actual code change for this looks fine, it's just that the PR is missing important context for me to be able to fully assess the scope of the change.

File: signbank/dictionary/models.py (hunk 1 above)
Comment (NavishkaJayasinghe) [reply signbank/dictionary/models.py]: Good suggestion. Will add that.

**Review by joshmcarthur — APPROVED** [review]
Nice!

**Review by Dimithri — APPROVED** [review]

**Pull request merged**